pandas/tests/extension/test_arrow.py

```diff
@@ -3553,3 +3553,10 @@ def test_categorical_from_arrow_dictionary():
         dtype="int64",
     )
     tm.assert_series_equal(result, expected)
+
+
+def test_arrow_json_type():
```

Comment:
You'll need to `@pytest.mark.skipif(pa_version_under19p0, reason=...)` as `pa.json_` is new as of Pyarrow 19